Source/Particles/Deposition/CurrentDeposition.H

```diff
@@ -2127,19 +2125,18 @@ void VillasenorDepositionShapeNKernel ([[maybe_unused]]amrex::ParticleReal const
  *        by segment, where the segments are determined by cell crossings. In general, this results
  *        in a tighter stencil. The implementation is valid for an arbitrary number of cell crossings.
  *
- * \param depos_order  deposition order
- * \param xp_n,yp_n,zp_n  Pointer to arrays of particle position at time level n.
+ * \tparam depos_order  deposition order
  * \param GetPosition  A functor for returning the particle position.
  * \param wp           Pointer to array of particle weights.
- * \param uxp_n,uyp_n,uzp_n  Pointer to arrays of particle momentum at time level n.
- * \param uxp_nph,uyp_nph,uzp_nph  Pointer to arrays of particle momentum at time level n + 1/2.
+ * \param uxp,uyp,uzp  Pointer to arrays of particle momentum.
  * \param ion_lev      Pointer to array of particle ionization level. This is
                        required to have the charge of each macroparticle
                        since q is a scalar. For non-ionizable species,
                        ion_lev is a null pointer.
  * \param Jx_arr,Jy_arr,Jz_arr  Array4 of current density, either full array or tile.
  * \param np_to_deposit         Number of particles for which current is deposited.
  * \param dt                    Time step for particle level
+ * \param relative_time         TODO
```

Comment:
@JustinRayAngus 

Does this docstring look correct to you?
```suggestion
 * \param relative_time         Time at which to deposit, relative to the time of the current positions of the particles.
```

Reply:
Adding the suggestion, further improvements or corrections can be done later on.